### ISSUE TITLE
will the project future support message queue like  rocketmq-rust?

### ISSUE THREAD

**Issue by dengyunsheng250** [spring-rs/spring-rs#20]
the repo is [here](https://github.com/mxsm/rocketmq-rust), and it seems keeping active

**Comment by holmofy** [comment]
Yes, [I am working on this](https://github.com/spring-rs/spring-rs/issues/3). I use [sea-streamer](https://github.com/SeaQL/sea-streamer), which abstracts the backend streaming. But it currently supports kafka and redis, as well as basic file and stdio. For kafka, I think [redpanda](https://github.com/redpanda-data/redpanda) can be used instead. It is a message middleware written in C++ and compatible with the kafka protocol, with higher performance and more reliability.

**Comment by holmofy** [comment]
This is a good suggestion. Friends in the community are welcome to submit a pull request to implement this plugin.
